src/products/src/products-service/data/products.yaml

```diff
@@ -430,4 +430,262 @@
   style: fishing
   description: Every tackle box needs a silver and red reflective fishing lure.
   price: 9.99
-  image: "12.jpg"
\ No newline at end of file
+  image: "12.jpg"
+- id: 1000567890
```

Comment:
One set of images were obtained with IDs from a separate AWS demo https://github.com/aws-samples/amazon-ivs-ecommerce-web-demo and we carried their IDs over.
The second set was numbered according to the video for which they were acquired. 
However, this can be changed for internal consistency if required.